src/sentry/static/sentry/app/components/resolutionBox.jsx

```diff
@@ -46,15 +47,29 @@ export default class ResolutionBox extends React.Component {
           />
         ),
       });
+    } else if (!!statusDetails.inCommit) {
+      return tct('This issue has been marked as resolved by [commit]', {
+        commit: (
+          <CommitLink
+            commitId={statusDetails.inCommit.id}
+            repository={statusDetails.inCommit.repository}
+          />
+        ),
+      });
     }
     return t('This issue has been marked as resolved.');
   };
 
   render = () => {
     return (
-      <div className="box">
-        <span className="icon icon-checkmark" />
-        <p className="truncate break-all">{this.renderReason()}</p>
+      <div
+        className="box"
+        style={{display: 'flex', alignItems: 'center', flex: 1, paddingBottom: 15}}
```

Comment:
This inline CSS could be moved into a styled components.

Reply:
Not worth the effort. It’s just more boilerplate as it’s a one-off.